public/src/components/sub_apps/admin.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Switch, Route, Redirect } from 'react-router-dom';
+
+// Containers:
+import CampaignsContainer from '../../containers/list_campaigns';
+import AdminQuestionsContainer from '../../containers/list_questions';
+import AdminScriptsContainer from '../../containers/list_scripts';
+import AdminContactListsContainer from '../../containers/list_contactlists';
+import EditAccountInfo from '../../components/account/edit_account';
+import CreateCampaignContainer from '../../containers/create_campaign';
+import CreateScriptContainer from '../../containers/create_script';
+import CreateQuestionContainer from '../../containers/create_question';
+import ViewQuestionContainer from '../../containers/view_question';
+import { ConnectedAccountPage } from '../../components/account/account';
+import ScriptPage from '../../containers/view_script';
+
+export default class Admin extends Component {
+  componentDidMount() {
+    const { id } = this.props.auth;
+    const { fetchUser } = this.props;
+    fetchUser(id);
+  }
+
+  render() {
+    const parent = '/admin';
+    return (
+      <Switch>
+        <Route
+          exact
+          path={`${parent}/questions/new`}
+          component={CreateQuestionContainer}
+        />
+        <Route
+          path={`${parent}/questions/:id`}
+          component={ViewQuestionContainer}
+        />
+        <Route
+          exact
+          path={`${parent}/scripts/new`}
+          component={CreateScriptContainer}
+        />
+        <Route
+          exact
+          path={`${parent}/account/:id/edit`}
```

Comment:
I think this route should live under /users/account/:id/edit for now. Best not to connect a view under 2 different routes? Same goes for the view account page below this one?

Reply:
Yeah, it's working for now, I wasn't sure if we were going to have an account view for admin. I can remove this if you think it's not needed here.